lib/actions/user.js

```diff
@@ -134,6 +136,8 @@ export function fetchOrInitializeUser (auth0User) {
 
     const isNewAccount = status === 'error' || (user && user.result === 'ERR')
     const userData = isNewAccount ? createNewUser(auth0User) : user
+    userData.language = 'fr'
+    userData.use24HourFormat = true
```

Comment:
These should probably not be hardcoded. It seems like something like this would be derived from the current locale setting that the user selected in the app.

Reply:
These were removed in https://github.com/opentripplanner/otp-react-redux/pull/375/commits/d0f962fcf847a7ba0d3b9a9b9fa33501ce41e00a.